### PR TITLE
feat: Examples

### PR DESCRIPTION
**Pull request by retraigo**
Examples:
- [x] Binary classification with Iris dataset.
- [x] Multiclass classification with iris dataset.
- [x] Binary text classification with SMS Spam Collection Dataset.
- [x] Multiple linear regression with student performance dataset.

Additional:
- [x] Expose the `scheduler`s to `mod.ts`.
- [x] Fix `log(0)` in `cross_entropy()` and division by zero in `cross_entropy_prime()`.

### CODE REVIEW

**Comment by retraigo** [comment]
There might still be something wrong with softmax / cross entropy. I'll make a commit later when I find it.

**Review by load1n9 — APPROVED** [review]
LGTM
